src/darwin/Framework/CHIP/MTRDeviceControllerLocalTestStorage.h

```diff
@@ -22,7 +22,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-MTR_EXTERN @interface MTRDeviceControllerLocalTestStorage : NSObject<MTRDeviceControllerStorageDelegate>
+MTR_EXTERN MTR_EXPORT @interface MTRDeviceControllerLocalTestStorage : NSObject<MTRDeviceControllerStorageDelegate>
```

Comment:
This is wrong anyway, being fixed here: https://github.com/project-chip/connectedhomeip/pull/32565/files?diff=split&w=0#diff-ade7f05b571936deec955f008edf7b4de285b29d9df637fd0fe25abba9456bc9